cafebabel/articles/models.py

```diff
@@ -63,9 +63,15 @@ def is_translated_in(self, language):
         given that it performs one query per language.
         TODO: turn into a cacheable list of translations when necessary.
         """
+        return bool(self.get_translation(language))
```

Comment:
I think forcing the `get` to only check the existence is a problem in terms of performances given that it actually retrieves the object for nothing.